mailbox/plugin/quota-search-elasticsearch/src/test/resources/quotaRatio.json

```diff
@@ -0,0 +1 @@
+{"user":"user@domain.org","domain":"domain.org","quotaRatio":0.55}
```

Comment:
do you really need a file for this?

Reply:
Avoid not carefull data formats CF recent PRs and @mbaechler  care about this.